src/openzaak/components/documenten/tests/test_lock_eio.py

```diff
@@ -157,11 +153,10 @@ def test_unlock_fail_incorrect_id(self):
         self.assertEqual(error['code'], 'incorrect-lock-id')
 
     def test_unlock_force(self):
-        self.autorisatie.scopes = [SCOPE_DOCUMENTEN_GEFORCEERD_UNLOCK]
-        self.autorisatie.save()
+        # self.autorisatie.scopes = [SCOPE_DOCUMENTEN_GEFORCEERD_UNLOCK]
+        # self.autorisatie.save()
```

Comment:
Isn't it better to skip this test, since we still want to test force unlock based on scope authorizations.